crowdsourcing/serializers/requester.py

```diff
@@ -2,10 +2,26 @@
 
 from crowdsourcing import models
 from rest_framework import serializers
+import datetime
 
 class RequesterSerializer(serializers.ModelSerializer):
+    total_projects = serializers.SerializerMethodField()
+    current_projects = serializers.SerializerMethodField()
+    waiting_projects = serializers.SerializerMethodField()
+
```

Comment:
it doesnt matter, what matters is the object type and these are a list of modules so they go under ModuleSerializer